framer/TextLayer.coffee

```diff
@@ -93,15 +93,27 @@ class exports.TextLayer extends Layer
 		super options
 		@__constructor = true
 
-		# Keeps track if the width or height are explicitly set, so we shouldn't update it afterwards
-		if not options.autoSize? and not options.truncate
+		# the goal is
+		# autoSize elements should not soft wrap, only hard wrap (based on newlines)
+		# fixed (given a box) elements should soft wrap
```

Comment:
Some weird newlines here, maybe add `- ` in front of the last two lines and a `:` after the first?